images/prow-tests/README.md

```diff
@@ -1,15 +1,45 @@
 # Prow Test Job Image
 
-This directory contains the custom Docker image used by our Prow test jobs. Some
-forks of this images exist under images/.
+This directory contains the custom Docker image used by our Prow test jobs.
 
 The `prow-tests` image is pinned on a specific `kubekins` image; update
 `Dockerfile` if you need to use a newer/different image. This will basically
 define the versions of `bazel`, `go`, `kubectl` and other build tools.
 
 The Prow jobs are configured to use the `prow-tests` image tagged with `stable`.
-This tag must be manually set in GCR using the Cloud Console.
 
 ## Building and publishing a new image
 
 See parent README.md
+
+## Testing and Deploying Images
+
+When `make push` is run for `prow-tests`, it publishes your image with the
+`:beta` label. At night Pacific time, a set of duplicate jobs are run using this
+new image and viewable at https://testgrid.knative.dev/beta-prow-tests
+
+A normal release process would involve first editing Dockerfile and/or programs
+it installs from test-infra, then running `make iterative-build` to build a
+local copy and ensure it still builds. Next, if adding a new tool or doing
+something complicated, do some rudimentary exploration in the image by running
+`make iterative-shell` to ensure it's working; most usage of prow-tests is via
+`runner.sh` so you might check a new version of Go by running
+`GO_VERSION=go1.30 runner.sh bash` and running `go version` or
+`go get something`.
+
+With an image you feel comfortable with deploying, create a PR to
+knative/test-infra and get approval. Once merged, pull upstream into your master
+branch and run `make push`. This will upload your image to the registry at
+http://gcr.io/knative-tests/test-infra/prow-tests with a date-commit_hash,
+`latest`, and `beta` tags. Let the image run at least a single overnight and
+ensure the jobs in the https://testgrid.knative.dev/beta-prow-tests testgrid are
```

Comment:
```suggestion
ensure the jobs in[testgrid](https://testgrid.knative.dev/beta-prow-tests) are
```

Here is an example

Reply:
There are multiple different testgrid dashboard groups being discussed and doing this adds about 50% more text when reading the markdown in the editor versus only removing 50% when viewing as a web page. That's why I'd prefer not to, but your call knowing my reasoning.

Reply:
We are certainly viewing this from different points of angles. I agree with you that adding these making code review harder, but this is one time thing vs. who knows how many times people go through this README, so I'm weighing more on the consumer side. 

However, this is not so big of a reason for blocking you, so 
/approve

Reply:
It's not about code review; when I'm editing Dockerfile I open the README.md in emacs at the same time. It highlights URLs so it looks good to me, so the `[text stuff]()` is just noise. Were I making a table of contents or if the particulars of the URLs weren't important, I'd definitely use the link bracket form. It just feels ambiguous here. I'll change if you want or I need /lgtm :-D